sdk/apps/modal-example/src/routes/solanaLazy.tsx

```diff
@@ -0,0 +1,117 @@
+import { createEffect, createSignal, onMount, Show } from 'solid-js'
+import { Title } from 'solid-start'
+import { NightlyConnectAdapter } from '@nightlylabs/wallet-selector-solana'
+import { Connection, PublicKey, SystemProgram, Transaction as SolanaTx } from '@solana/web3.js'
+import toast from 'solid-toast'
+
+const connection = new Connection('https://api.devnet.solana.com')
+
+export default function SolanaLazy() {
+  const [adapter, setAdapter] = createSignal<NightlyConnectAdapter>()
+  const [eagerConnect, setEagerConnect] = createSignal(false)
+  const [initialized, setInitialized] = createSignal(false)
+  const [publicKey, setPublicKey] = createSignal<PublicKey>()
+  onMount(() => {
+    const adapter = NightlyConnectAdapter.buildLazy(
+      {
+        appMetadata: {
+          name: 'NCTestSolana',
+          description: 'Nightly Connect Test',
+          icon: 'https://docs.nightly.app/img/logo.png',
+          additionalInfo: 'Courtesy of Nightly Connect team'
+        },
+        url: 'https://nc2.nightly.app'
+      },
+      true,
+      document.getElementById('modalAnchor') ?? undefined
```

Comment:
do we need it ?

Reply:
getElementById returns null, but I can extend anchorRef type for that case

Reply:
fixed